src/seedu/addressbook/commands/Command.java

```diff
@@ -39,12 +39,12 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
+/*    public CommandResult execute(){
```

Comment:
it is better to remove unnecessary code completely, rather than commenting it out.
you can always retrieve it from VCS.